libchisel/src/imports.rs

```diff
@@ -21,6 +50,28 @@ impl<'a> ImportList<'a> {
     pub fn with_entries(entries: Vec<ImportType<'a>>) -> Self {
         ImportList(entries)
     }
+
+    pub fn lookup_by_field(&self, name: &str) -> Option<&ImportType> {
+        let entries = self.entries();
+
+        for import in entries {
+            match import {
```

Comment:
Cannot this use `import.field() == name` ?

Reply:
will refactor, this was written before the other helpers